deploy/us-east-2-dev/config.yaml

```diff
@@ -0,0 +1,66 @@
+data:
+  config: |
+    {
+      "Identity": {
+        "PeerID": "12D3KooWBkPTEgcCmBGgN1VgxjdQrYKeGA5CFQ7W5gP6bBJYp9WF",
+        "PrivKey": "$STORETHEINDEX_PRIVATE_KEY"
+      },
+      "Addresses": {
+        "Admin": "/ip4/0.0.0.0/tcp/3002",
+        "Finder": "/ip4/0.0.0.0/tcp/3000",
+        "Ingest": "/ip4/0.0.0.0/tcp/3001",
+        "P2PAddr": "/ip4/0.0.0.0/tcp/3003"
+      },
+      "Bootstrap": {
+        "Peers": [
+          "/dns4/bootstrap-0.ipfsmain.cn/tcp/34721/p2p/12D3KooWQnwEGNqcM2nAcPtRR9rAX8Hrg4k9kJLCHoTR5chJfz6d",
+          "/dns4/bootstrap-3.mainnet.filops.net/tcp/1347/p2p/12D3KooWKhgq8c7NQ9iGjbyK7v7phXvG6492HQfiDaGHLHLQjk7R",
+          "/dns4/bootstrap-4.mainnet.filops.net/tcp/1347/p2p/12D3KooWL6PsFNPhYftrJzGgF5U18hFoaVhfGk7xwzD8yVrHJ3Uc",
+          "/dns4/bootstrap-6.mainnet.filops.net/tcp/1347/p2p/12D3KooWP5MwCiqdMETF9ub1P3MbCvQCcfconnYHbWg6sUJcDRQQ",
+          "/dns4/bootstrap-7.mainnet.filops.net/tcp/1347/p2p/12D3KooWRs3aY1p3juFjPy8gPN95PEQChm2QKGUCAdcDCC4EBMKf",
+          "/dns4/bootstrap-8.mainnet.filops.net/tcp/1347/p2p/12D3KooWScFR7385LTyR4zU1bYdzSiiAb5rnNABfVahPvVSzyTkR",
+          "/dns4/node.glif.io/tcp/1235/p2p/12D3KooWBF8cpp65hp2u9LK5mh19x67ftAam84z9LsfaquTDSBpt",
+          "/dns4/bootstrap-2.mainnet.filops.net/tcp/1347/p2p/12D3KooWEWVwHGn2yR36gKLozmb4YjDJGerotAPGxmdWZx2nxMC4",
+          "/dns4/bootstrap-5.mainnet.filops.net/tcp/1347/p2p/12D3KooWLFynvDQiUpXoHroV1YxKHhPJgysQGH2k3ZGwtWzR4dFH",
+          "/dns4/bootstrap-1.starpool.in/tcp/12757/p2p/12D3KooWQZrGH1PxSNZPum99M1zNvjNFM33d1AAu5DcvdHptuU7u",
+          "/dns4/bootstrap-0.mainnet.filops.net/tcp/1347/p2p/12D3KooWCVe8MmsEMes2FzgTpt9fXtmCY7wrq91GRiaC8PHSCCBj",
+          "/dns4/bootstrap-1.mainnet.filops.net/tcp/1347/p2p/12D3KooWCwevHg1yLCvktf2nvLu7L9894mcrJR4MsBCcm4syShVc",
+          "/dns4/bootstrap-0.starpool.in/tcp/12757/p2p/12D3KooWGHpBMeZbestVEWkfdnC9u7p6uFHXL1n7m1ZBqsEmiUzz",
+          "/dns4/lotus-bootstrap.ipfsforce.com/tcp/41778/p2p/12D3KooWGhufNmZHF3sv48aQeS13ng5XVJZ9E6qy2Ms4VzqeUsHk",
+          "/dns4/bootstrap-1.ipfsmain.cn/tcp/34723/p2p/12D3KooWMKxMkD5DMpSWsW7dBddKxKT7L2GgbNuckz9otxvkvByP"
+        ],
+        "MinimumPeers": 15
+      },
+      "Datastore": {
+        "Type": "levelds",
+        "Dir": "datastore"
+      },
+      "Discovery": {
+        "LotusGateway": "wss://api.chain.love",
+        "Policy": {
+          "Allow": true,
+          "Except": null,
+          "Trust": true,
+          "TrustExcept": null
+        },
+        "PollInterval": "24h0m0s",
+        "RediscoverWait": "5m0s",
+        "Timeout": "2m0s"
+      },
+      "Indexer": {
+        "CacheSize": 300000,
+        "ValueStoreDir": "valuestore",
+        "ValueStoreType": "sth"
+      },
+      "Ingest": {
+        "PubSubTopic": "/indexer/ingest/mainnet",
+        "StoreBatchSize": 256,
+        "SyncTimeout": "2h0m0s",
+        "EntriesDepthLimit": 400,
+        "AdvertisementDepthLimit": 400
+      }
+    }
```

Comment:
It may be clearer to:
1. Define env vars in code and use configmap `envFrom` in deployment instead of mounting config.
    This will have the added advantage of making overrides easier when we have a base overlay and a concrete overlay.
3. Remove any default values.